src/bll/selectors/dailyTable.ts

```diff
@@ -0,0 +1,5 @@
+import { AppState } from "../store";
+
```

Comment:
Тут `dailyTable` - это динамическое название, лучше использоввать `[dailyTableSlice.name]` + вынести в отдельный селектор извлечение стейта:
```suggestion
export const getDailyTable = state[dailyTableSlice.name];
```
и переиспользовать его